### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by howardjones**
These files work with tesseract 5.0.0 too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 tessdata
 ========
 
-These language data files only work with Tesseract 4.0.0.
+These language data files only work with Tesseract 4.0.0 or 5.0.0-alpha.
```
Comment (stweil) [inline README.md:4]:
```suggestion
These language data files only work with Tesseract 4.0.0 an newer versions.
```

Comment (stweil) [reply README.md]: They also work with Tesseract 4.1.0, so "newer versions" is better than adding a lengthy list of versions here.

Comment (howardjones) [reply README.md:4]: Sure, if that is accurate...? I only have two data points. (this is why I shouldn't submit docs updates, too - only one point of view! :-) )

**Review by stweil — CHANGES_REQUESTED** [review]

**Review by stweil — APPROVED** [review]
Thanks.

**Pull request merged**